pkg/compact/downsample/downsample_test.go

```diff
@@ -752,15 +753,14 @@ func TestCounterSeriesIteratorSeek(t *testing.T) {
 	var res []sample
 	x := NewApplyCounterResetsIterator(its...)
 
-	ok := x.Seek(150)
-	testutil.Assert(t, ok, "Seek should return true")
+	valueType := x.Seek(150)
+	testutil.Equals(t, chunkenc.ValFloat, valueType, "Seek should return true")
```

Comment:
Nit: let's update the message here:
```suggestion
	testutil.Equals(t, chunkenc.ValFloat, valueType, "Seek should return float value type")
```